### ISSUE TITLE
[win][v469a] stat fps command font's too small size + bad mixed color makes it unreadable

### ISSUE THREAD

**Issue by gargul2** [OldUnreal/UnrealTournamentPatches#59]
Due to lack of asnwer for Issue 1 in :https://github.com/OldUnreal/UnrealTournamentPatches/issues/49#issuecomment-699619351 I report it then.

**Comment by SeriousBuggie** [comment]
This issue happens to all debug output. Especially all `STAT` commands. `STAT GLOBAL` another one.

**Comment by racing-online** [comment]
Solution would be to use non-transparent font draw style with black-colored background.

**Comment by Chamberly** [comment]
I see it pretty fine on my end but I'm a big fan of small fonts. Here is what I got in 800x600. It looks exactly the same to me with v436 as well.
![image](https://user-images.githubusercontent.com/11419314/94448160-97d5b300-0178-11eb-9a37-342803dd3b89.png)
But yes when it come to looking at it in umenu, that's a different story. I just thought hey it's normal, no biggie cuz I see it in game render frame and that was my attention was at.
![image](https://user-images.githubusercontent.com/11419314/94448550-0286ee80-0179-11eb-9311-956e60d3ea04.png)

**Comment by SeriousBuggie** [comment]
HUD can be bright in game too.
Look at this:
![scr_1601306772](https://user-images.githubusercontent.com/70026933/94452574-5c5ad900-01b8-11eb-85e7-95ee721c2a07.png)
Typical in MH. Need reduce HUD for see FPS line:
![scr_1601306812](https://user-images.githubusercontent.com/70026933/94452589-61b82380-01b8-11eb-8646-51bf2447e050.png)

**Comment by gargul2** [comment]
> Chamberly : https://github.com/OldUnreal/UnrealTournamentPatches/issues/59#issuecomment-700060040 : I see it pretty fine on my end but I'm a big fan of small fonts. Here is what I got in 800x600. It looks exactly the same to me with v436 as well.

Good joke, not representative at all, no wonder at such a extreme low antique resolution like 800x600, now let's go back from the past to the present - https://store.steampowered.com/hwsurvey clearly shows that 1920x1080 dominates and we should stick around it as a main game resolution, with taking into account popularizing 2K and 4k

![1](https://user-images.githubusercontent.com/63287518/94459498-b7400080-01b7-11eb-9bfa-b190b75c3d24.png)

**Comment by racing-online** [comment]
Some players need as high FPS as possible in order to minimize latency... Looks like they are able to recognize latency intervals below 16ms. For that reason they also use reduced screen resolutions.

**Comment by gargul2** [comment]
You can generate 400 FPS with 2.5 ms latency, but still your 60 Hz monitor will display only 60 FPS of them with 16ms latency, and 340 FPS will be just dropped: https://www.avadirect.com/blog/frame-rate-fps-vs-hz-refresh-rate/

**Comment by racing-online** [comment]
Yes, exactly. There's one problem with the latency though - once frame rendering is completed, it must wait for the screen refresh time until rendering of the next frame will begin. This wait interval with 60Hz screen refresh rate can reach up to ~16ms. This is the reason that produces additional latency between the input - mouse, keyboard, etc., and the output - picture user sees on the screen.
When frames are being rendered with higher rates than the screen refresh rate, this latency is reduced. For instance, if for simplicity to omit other factors affecting latency, with rendering frame rate of 400 FPS latency in theory wouldn't reach over than 2.5ms. Of course in this case penalty of this is the picture tearing, since the rendering process isn't synchronized with the screen refresh rate incomplete frames appear on the screen where part of the picture is from the currently drawing frame and other part is from the previous one. Another penalty of this is a lot of wasted GPU time for rendering of extra frames the player never sees, as you already mentioned.

So this is the matter of personal preferences - whether to choose high quality without tearing by the cost of high latency, or minimized latency by the cost of tearing with low picture quality.
There are modern synchronization methods which eliminate this problem, but they are hardware dependent and aren't available for everyone.

**Comment by gargul2** [comment]
OK, I see now, I have never had problems with hitting my opponents on 60 FPS anyway, and such settings like 400 FPS & 800x600 are stille exterme, I would never in my life resign from native res and accept screen tearing, the best choice seems to be to buy a 120 / 144 / 240 Hz Monitor together with a GPU which can generate 120 / 144 / 240 FPS with native res (FullHD or 4K). Ths guy doesn't care about native res quality so most probably he doesn't use stuff like 4K textures, HD skins, 4K resolution either, and even with such stuff some strong GPU can handle such old game.

**Comment by Chamberly** [comment]
Stay on topic please.

**Comment by gargul2** [comment]
OK, before even begin to off-topic I forgot to ask you whether 800x600 & 400 FPS are settings you actually use to play..., perhaps you just wanted to show that font looks ok on low res as I've realised just now that you used `setres` command, the same apply for 400 FPS maybe you were just doing some performance or performance issue testing. OH my god I'm a deer.

**Comment by gargul2** [comment]
Sure, if you just wouldn't post screenshoots with some crappy weird settings like 800x600 & 400 FPS an off-topic would not even begin.

**Comment by gargul2** [comment]
Beware troll Chamberly, he has blacklisted me on github.
